src/app/excel/excel-table/excel-table.component.ts

```diff
@@ -75,7 +80,7 @@ export class ExcelTableComponent implements OnInit {
   }
 
   closeSprint() {
-
+      //TODO: dodać wysyłanie posta z zamknięciem sprintu
```

Comment:
W tym widoku ma być tylko przycisk _Zamknij sprint_, w jego funkcji `onClick()` nie masz wysłać posta o zamknięcie sprintu, tylko przekierować na widok zamykania sprintu (`/sprints/close-sprint`).

Reply:
okej to trochę upraszcza